src/main.ts

```diff
@@ -0,0 +1,89 @@
+import { Command, Helper } from 'dojo-cli/interfaces';
+import { Argv } from 'yargs';
+import * as fs from 'fs';
+import * as path from 'path';
+const DtsCreator: any = require('typed-css-modules');
+const read: any = require('read-file-stdin');
+const globby: any = require('globby');
```

Comment:
I believe that there are typings available for most of these, can you `import` them instead please?